package.json

```diff
@@ -14,37 +14,43 @@
     "preview": "vite preview",
     "serve": "vite build && vite preview",
     "test": "playwright test",
+    "test:smoke": "playwright test tests/smoke.test.ts",
```

Comment:
looks like boilerplate to me, just use `pnpm test --headed` or `pnpm test tests/smoke.test.ts`, no need to define new scripts for that